components/function-controller/README.md

```diff
@@ -0,0 +1,176 @@
+# Function Controller
+
+The Knative Function Controller is a Kubernetes controller that enables Kyma to manage Function resources. It uses Knative Build and Knative Serving under the hood.
+
+## Prerequisites
+
+The Function Controller requires the following components to be installed:
+
+- Knative Build (v0.6.0)
+- Knative Serving (v0.6.1)
+- Istio (v1.0.7)
+
```

Comment:
```suggestion
## Installation
```
Sorry for messing with the headings :D but we can group the next two under Installation.